code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -510,6 +510,48 @@
 		new /mob/living/simple_animal/pet/dog/corgi(location)
 	..()
 
+//monkey powder heehoo
+/datum/chemical_reaction/monkey_powder
+	name = /datum/reagent/monkey_powder
+	id = /datum/reagent/monkey_powder
+	results = list(/datum/reagent/monkey_powder = 3)
+	required_reagents = list(/datum/reagent/consumable/banana = 1, /datum/reagent/consumable/nutriment=2,/datum/reagent/liquidgibs = 1)
+
+/datum/chemical_reaction/monkey
+	name = "monkey"
+	id = "monkey"
+	required_reagents = list(/datum/reagent/monkey_powder = 30, /datum/reagent/water = 1)
+
+/datum/chemical_reaction/monkey/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	new /mob/living/carbon/monkey(location)
+//water electrolysis
+/datum/chemical_reaction/electrolysis
+	name = "electrolysis"
+	id = "electrolysis"
+	results = list(/datum/reagent/oxygen = 10, /datum/reagent/hydrogen = 20)
+	required_reagents = list(/datum/reagent/consumable/liquidelectricity = 1, /datum/reagent/water = 5)
+
+//butterflium
+/datum/chemical_reaction/butterflium
+	name = "butterflium"
+	id = "butterflium"
+	required_reagents = list(/datum/reagent/colorful_reagent = 1, /datum/reagent/medicine/omnizine = 1, /datum/reagent/medicine/strange_reagent = 1, /datum/reagent/consumable/nutriment = 1)
+/datum/chemical_reaction/butterflium/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = rand(1, created_volume), i <= created_volume, i++)
+		new /mob/living/simple_animal/butterfly(location)
+	..()
+//scream powder
+/datum/chemical_reaction/scream
+	name = "scream"
+	id = "scream"
+	required_reagents = list(/datum/reagent/medicine/strange_reagent = 1, /datum/reagent/consumable/cream = 5, /datum/reagent/consumable/ethanol/lizardwine = 5	)
+	required_temp = 374
+
+/datum/chemical_reaction/scream/on_reaction(datum/reagents/holder, created_volume)
+	playsound(holder.my_atom, pick(list( 'sound/voice/human/malescream_1.ogg', 'sound/voice/human/malescream_2.ogg', 'sound/voice/human/malescream_3.ogg', 'sound/voice/human/malescream_4.ogg', 'sound/voice/human/malescream_5.ogg', 'sound/voice/human/malescream_6.ogg', 'sound/voice/human/femalescream_1.ogg', 'sound/voice/human/femalescream_2.ogg', 'sound/voice/human/femalescream_3.ogg', 'sound/voice/human/femalescream_4.ogg', 'sound/voice/human/femalescream_5.ogg', 'sound/voice/human/wilhelm_scream.ogg')), created_volume*5,TRUE)
```

Comment:
uh, what happens if you make, like, 100u of this react with someone at once? 1000u?

Also, why not just make this use a creature's "natural" scream sound? Why would a moth scream like a human when sprayed with scream powder? Also, since most non-humanoids don't have scream sounds (I think), this lowers the potential for just spamming a SHITTON of scream noises by detonating a smoke grenade with scream powder in it near a stack of, like, 50 simplemobs.

Reply:
im pretty sure sounds have don’t go beyond 100%? not totally sure i’ve just seen sounds with 300 that sound no different than 100

Reply:
it is capped at 100 iirc as well as but since it's a real pain to get i'm not scared of anyone abusing it
and if they do it might make for a really good story